docs/sources/operators-guide/monitor-grafana-mimir/dashboards/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 aliases:
-  - /../visualizing-metrics/dashboards/
+  - ../visualizing-metrics/dashboards/
```

Comment:
This is relative from what Hugo regards as the root. From my understanding, without it, this is relative to the file.